overthebox/files/etc/uci-defaults/otb-network.defaults

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+# vim: set noexpandtab tabstop=4 shiftwidth=4 softtabstop=4 :
+
+uci -q batch <<EOF
+set network.lan.proto=static
+set network.lan.ipaddr=192.168.100.1
```

Comment:
Yep that was only a fix for the current master,
We'll do a specific PR for the v1 migration.